packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -237,6 +248,10 @@ export default {
         fees: {
             type: Object,
             default: () => {}
+        },
+        availability: {
+            type: Object,
+            default: null
         }
```

Comment:
fees and availability are both objects but have two different defaults - could availability's default also be default: () => {} or default: () => ({})?

Reply:
I'd prefer the default to be null so we can simply not render the component if the entire data object is missing. I'm doing to default most of them to null in a future ticket!

Reply:
Ok cool 👍